ucm2/conf.d/macaudio/laptops/4spk/HiFi.conf

```diff
@@ -0,0 +1,56 @@
+# For laptops with 4 speakers (4 woofers)
+
+SectionVerb {
+	EnableSequence [
+		cset "name='Headphone Playback Mux' Primary"
+		cset "name='Speaker Playback Mux' Secondary"
+	]
+
+	Value {
+		TQ "HiFi"
+	}
+}
+
+Define.JackDevice "0"
+If.cs42l84 {
+	Condition {
+		Type ControlExists
+		Control "name='Jack ADC WNF Switch'"
+	}
+	True {
+		Include.hp.File "/conf.d/macaudio/cs42l84/Headphones.conf"
+	}
+}
+
+If.cs42l83 {
+	Condition {
+		Type ControlExists
+		Control "name='Jack ADC Boost Switch'"
+	}
+	True {
+		Include.hp.File "/conf.d/macaudio/cs42l83/Headphones.conf"
+	}
+}
+
+
+If.havespk {
+	Condition {
+		Type ControlExists
+		Control "name='Left Woofer Speaker Volume'"
```

Comment:
this doesn't work for all devices. j293 and j493 have "Left Front" and "Left Rear". The control for j293 is `Speaker Playback Volume`. I'd recommend using a define for this in the device `.conf` files